trdg/data_generator.py

```diff
@@ -174,6 +174,26 @@ def generate(
             "RGB", (background_width, background_height), (0, 0, 0)
         )
 
+        ##############################################################
+        # Comparing average pixel value of text and background image #
+        ##############################################################
+        try:
+            resized_img_st = ImageStat.Stat(resized_img, resized_mask.split()[2])
+            background_img_st = ImageStat.Stat(background_img) 
+
+            resized_img_px_mean = sum(resized_img_st.mean[:2]) / 3
```

Comment:
`resized_img_st.mean` cannot be calculated if `resized_img_st.count = [0, 0, 0, 0]` because that causes a division by zero. This case occurs in my tests when running `trdg -c 1 -w 5 -f 64 -hw`.